rollup.config.js

```diff
@@ -0,0 +1,52 @@
+import babel from 'rollup-plugin-babel';
+import resolve from 'rollup-plugin-node-resolve';
+import commonJS from 'rollup-plugin-commonjs';
+import json from 'rollup-plugin-json';
+import builtins from 'rollup-plugin-node-builtins';
+
+const output = (name, format) => ({
+  name,
+  file: `dist/webexWidgets.${format}.js`,
+  format,
+  sourcemap: true,
+  globals: {
+    bufferutil: 'bufferutil',
+    'prop-types': 'PropTypes',
+    react: 'React',
+    'react-dom': 'ReactDOM',
+    'spawn-sync': 'spawnSync',
```

Comment:
What does this do?

Reply:
global dependency from webex module. needs to be externally declared.

Reply:
I’m not familiar with this global dependency, can you link me to it so maybe we can remove it. We don’t want anyone else with the Webex module to have to do this.

Reply:
![Screen Shot 2020-02-10 at 9 38 25 AM](https://user-images.githubusercontent.com/14226615/74174620-17624200-4be9-11ea-941d-d20bdec753a0.png)
webex might not utilize this library directly since I cant find inside the repo. However, these warnings disappear when I exclude the webex form the bundling process.

Reply:
not trying to reason my approach. but this is what we have in [sdk-adpter](https://github.com/webex/sdk-component-adapter/blob/master/rollup.config.js#L59) rollup configuration too.

Reply:
We need to have a tech debt item to figure out what is causing this. IMO this is a workaround and not a solution.